http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -863,7 +871,12 @@ class DataprocInterpreter[F[_]: Parallel](
     val dataprocProps = if (machineConfig.numberOfWorkers == 0) {
       // Set a SoftwareConfig property that makes the cluster have only one node
       Map("dataproc:dataproc.allow.zero.workers" -> "true")
-    } else Map.empty[String, String]
+    } else {
+      getDataProcMachineSpecificationFromMachineTypeName(machineConfig.workerMachineType) match {
```

Comment:
For some reason I've been assuming this configuration is for the worker machines, but now after rereading the comments in the ticket, I think this is actually for the master machine?

Reply:
Ahhhhh driver == "master"